index.html

```diff
@@ -1877,11 +1891,12 @@ <h3>Securing Verifiable Credentials</h3>
         <p class="note">
 As discussed in Section <a href="#conformance"></a>, there are multiple viable
 proof mechanisms, and this specification does not standardize nor recommend any
-single proof mechanism for use with <a>verifiable credentials</a>. For more
-information about the <code>proof</code> mechanism, see the following
-specifications: Data Integrity [[VC-DATA-INTEGRITY]], Securing Verifiable
-Credentials using JSON Web Tokens [VC-JWT], and the "Proofs" section of
-the Verifiable Credential Specifications Directory [[VC-SPECS]].
+single proof mechanisms for use with <a>verifiable credentials</a> or
+<a>verifiable presentations</a>. For more information about the
+<code>proof</code> mechanisms, see the following specifications: Data Integrity
+[[VC-DATA-INTEGRITY]], Securing Verifiable Credentials using JSON Web Tokens
+[VC-JWT], and the "Proofs" section of the Verifiable Credential Specifications
+Directory [[VC-SPECS]].
```

Comment:
I'd like to note that vc-jose-cose will be added to [a different section](https://github.com/w3c/vc-specs-dir/pull/14) in the specs dir, so this language should probably be more generic....or the specs dir should be reorganized to include all embedded and non embedded securing mechanisms in a similar section cc: @OR13

Reply:
imo, sorta blocked by https://github.com/w3c/vc-specs-dir/pull/14